x-pack/plugins/fleet/server/services/agents/agentless_agent.ts

```diff
@@ -112,20 +114,22 @@ class AgentlessAgentService {
         }
         if (error.response) {
           logger.error(
-            `Creating agentless failed with a response status code that falls out of the range of 2xx: ${error.response.status} ${error.response.statusText} ${requestConfig.data}`
+            `Creating agentless failed with a response status code that falls out of the range of 2xx: ${
+              error.response.status
```

Comment:
why are we keep statusText message? Does  error.response.status  and error.response.statusText have the same message?

Reply:
Should we capture `error.message` here?

Reply:
> why are we keep statusText message? Does error.response.status and error.response.statusText have the same message?

I removed the `statusText` because it essentially describes `response.status`

> Should we capture error.message here?

If the type of the error is `AxiosError` there is no `error.message`,  we need to use `error.cause` however in this case since the `error.response` is present we should show the error response status and data